content/en/blog/2025/pqc-in-k8s/pqc-in-k8s.md

```diff
@@ -0,0 +1,195 @@
+---
+layout: blog
+title: "Post-Quantum Cryptography in Kubernetes"
+slug: pqc-in-k8s
+date: 2025-06-03
+author: "Fabian Kammel (ControlPlane)"
+---
+
+The world of cryptography is on the cusp of a major shift with the advent of
+quantum computing. While powerful quantum computers are still largely
+theoretical for many applications, their potential to break current
+cryptographic standards is a serious concern, especially for long-lived
+systems. This is where _Post-Quantum Cryptography_ (PQC) comes in. In this
+article, I\'ll dive into what PQC means for TLS and, more specifically, for the
```

Comment:
Is it actually necessary to escape apostrophes?  This doesn't break anything, but I don't recall it being required before.